app/helpers/publishers_helper.rb

```diff
@@ -231,7 +231,23 @@ def statement_period_description(period)
   end
 
   def statement_period_date(date)
-    date.strftime('%B %e, %Y')
+    date.strftime('%B %e')
+  end
+
+  def link_to_most_recent_statement
+    most_recent_statement = current_publisher.statements.order(created_at: :desc).first
+    if most_recent_statement
```

Comment:
Nit pick: I prefer using `most_recent_statement.present?`, even though the results are the same. The reviewer has to be certain that the type is not a collection, otherwise an empty collection would enter this block.

Reply:
Good call 👍 I just amended that commit.